kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/DisbursementVoucherNonEmployeeTravel.xml

```diff
@@ -0,0 +1,43 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:p="http://www.springframework.org/schema/p"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans 
+		http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+	<bean id="DisbursementVoucherNonEmployeeTravel-disbVchrServicePerformedDesc-parentBean" abstract="true" parent="AttributeDefinition">
```

Comment:
We should override the child bean instead of the parent bean and update the label and shortlabel fields.  The other fields should pull from the parent bean.  Look at edu.arizona.kfs.fp.businessobject.datadictionary.ProcurementCardHolder.xml.